<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="66" started-at="2023-08-01T20:56:49 IST" finished-at="2023-08-01T20:56:49 IST">
    <groups>
      <group name="SmokeSuite">
        <method signature="CreateContactWithOrganizationTest.createContactWithOrgTest()[pri:0, instance:Vtiger.Contacts.test.CreateContactWithOrganizationTest@2eda0940]" name="createContactWithOrgTest" class="Vtiger.Contacts.test.CreateContactWithOrganizationTest"/>
      </group> <!-- SmokeSuite -->
    </groups>
    <test name="Default test" duration-ms="66" started-at="2023-08-01T20:56:49 IST" finished-at="2023-08-01T20:56:49 IST">
      <class name="Vtiger.Contacts.test.CreateContactWithOrganizationTest">
        <test-method status="PASS" signature="bsConfig()[pri:0, instance:Vtiger.Contacts.test.CreateContactWithOrganizationTest@2eda0940]" name="bsConfig" is-config="true" duration-ms="0" started-at="2023-08-01T20:56:49 IST" finished-at="2023-08-01T20:56:49 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bsConfig -->
        <test-method status="FAIL" signature="bcConfig()[pri:0, instance:Vtiger.Contacts.test.CreateContactWithOrganizationTest@2eda0940]" name="bcConfig" is-config="true" duration-ms="0" started-at="2023-08-01T20:56:49 IST" finished-at="2023-08-01T20:56:49 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at Vtiger.GenericUtilities.BaseClass.bcConfig(BaseClass.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bcConfig -->
        <test-method status="SKIP" signature="bmConfig()[pri:0, instance:Vtiger.Contacts.test.CreateContactWithOrganizationTest@2eda0940]" name="bmConfig" is-config="true" duration-ms="-1690903609906" started-at="2023-08-01T20:56:49 IST" finished-at="1970-01-01T05:30:00 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bmConfig -->
        <test-method status="SKIP" signature="createContactWithOrgTest()[pri:0, instance:Vtiger.Contacts.test.CreateContactWithOrganizationTest@2eda0940]" name="createContactWithOrgTest" duration-ms="0" started-at="2023-08-01T20:56:49 IST" finished-at="2023-08-01T20:56:49 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at Vtiger.GenericUtilities.BaseClass.bcConfig(BaseClass.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createContactWithOrgTest -->
        <test-method status="SKIP" signature="amConfig()[pri:0, instance:Vtiger.Contacts.test.CreateContactWithOrganizationTest@2eda0940]" name="amConfig" is-config="true" duration-ms="-1690903609922" started-at="2023-08-01T20:56:49 IST" finished-at="1970-01-01T05:30:00 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- amConfig -->
        <test-method status="SKIP" signature="acConfig()[pri:0, instance:Vtiger.Contacts.test.CreateContactWithOrganizationTest@2eda0940]" name="acConfig" is-config="true" duration-ms="-1690903609922" started-at="2023-08-01T20:56:49 IST" finished-at="1970-01-01T05:30:00 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- acConfig -->
        <test-method status="PASS" signature="asConfig()[pri:0, instance:Vtiger.Contacts.test.CreateContactWithOrganizationTest@2eda0940]" name="asConfig" is-config="true" duration-ms="0" started-at="2023-08-01T20:56:49 IST" finished-at="2023-08-01T20:56:49 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- asConfig -->
      </class> <!-- Vtiger.Contacts.test.CreateContactWithOrganizationTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
